interface/views.py

```diff
@@ -74,6 +75,26 @@ def homepage(request):
                    'logout_url': redirect(logout_view).url})
 
 
+@login_required
+def review(request, pk):
+    if request.POST and request.user.is_staff:
```

Comment:
You should check for `is_staff` at the top of the view, using the [staff_member_required](https://docs.djangoproject.com/en/2.2/ref/contrib/admin/#django.contrib.admin.views.decorators.staff_member_required) decorator. Also, the view should only take `POST` requests, and you can do this with the [require_http_methods](https://docs.djangoproject.com/en/2.2/topics/http/decorators/#django.views.decorators.http.require_http_methods) decorator.